### ISSUE TITLE
Selecting multiple research data (sub) foldersto publish [Feature request]

### ISSUE THREAD

**Issue by davetromp** [Sciebo-RDS/rds-ng#12]
In our pilot of ScieboRDS, we received following feedback, that you might want to consider while develing the NG version.

Sciebo RDS requires you to select the folder(s) to be published to the repository. It is possible to select one project folder.

It would be nice if it was possible to select several subfolders of the project folder at this stage (e.g. a raw data subfolder and a final data subfolder). This would allow multiple subfolders to be uploaded at the same time.

So basically the user expresses the wish to be able to select multiple folders a la carte and create a project this way.
